cli/install/template.go

```diff
@@ -788,7 +803,7 @@ spec:
         - "-log-level={{.ControllerLogLevel}}"
         ports:
         - name: proxy-injector
-          containerPort: 443
+          containerPort: 8443
```

Comment:
👍 Great, looks like this fixes the issue that we were having with this container.